### PR TITLE
change label of getLocation button in geopoint widget

### LINKED ISSUE #1859: Inconsistency between Geopoint widgets' button name
#### Software and hardware versions 
Collect v1.12.2 and older version

#### Problem description
The button text changes if any location is recorded if we use the `GeopointWidget` with `map` appearance but it doesn't if we use that widget without any appearance

screenshot_2018-02-13-13-58-13
screenshot_2018-02-13-13-58-15

#### Steps to reproduce the problem
You can use `All Widgets` form.

#### Expected behavior
I think the text in the first case should be changed too. Maybe `View or Change Location` is not good since we can't view that location but we can use eg. just `Change Location`

#### Other information 
The problem is visible if we use both Google and OSM SDK.

@opendatakit-bot label "quick win"

### PR DESCRIPTION
**Pull request by SIMRAN1**
Closes #fixes #1859

### CODE REVIEW

**Comment by grzesiek2010** [comment]
@SIMRAN1 Could you fix your build?

File: collect_app/src/main/java/org/odk/collect/android/widgets/GeoPointWidget.java [hunk 1]
```diff
@@ -157,8 +157,12 @@ private void updateButtonLabelsAndVisibility(boolean dataAvailable) {
                 getLocationButton.setVisibility(View.GONE);
             } else {
                 getLocationButton.setVisibility(View.VISIBLE);
-                getLocationButton.setText(getContext().getString(
-                        dataAvailable ? R.string.get_point : R.string.get_point));
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/widgets/GeoPointWidget.java:161]: I believe you can get the desired behavior just by changing the first `R.string.get_point` to your new `R.string.change_location`! It looks like `dataAvailable` is equivalent to your test to see if there is an answer already. If you're not familiar with this syntax, it's equivalent to a conditional: https://www.sitepoint.com/java-ternary-operator/

Comment (lognaturel) [reply collect_app/src/main/java/org/odk/collect/android/widgets/GeoPointWidget.java:161]: @SIMRAN1 Does the above make sense? Please ask if you have any questions.

**Comment by lognaturel** [comment]
Closing so someone else can take the issue up.

**Pull request closed without merging**